frontend/src/components/common/PieChart.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+    <VContainer>
+        <VSubheader
+            class="header my-3 text-dark"
+            fluid
+        >
+            Summary
+        </VSubheader>
+        <VLayout class="pie-container">
+            <VFlex
+                lg4
+                md4
+                hidden-sm-and-down
+                height="100%"
+                class="img-card"
+            >
+                <GChart
+                    type="PieChart"
+                    :data="chartData"
+                    :options="chartOptions"
+                />
+            </VFlex>
+            <VFlex
+                lg5
+                md5
+                hidden-sm-and-down
+                height="100%"
+                class="img-card"
+            >
+                <VSubheader
+                    v-text="legend.title"
+                    class="legend-title text-dark"
+                />
+                <VList>
+                    <VListItem
+                        v-for="visitor in legend.data"
+                        :key="visitor.title"
+                    >
+                        <VRow class="align-center justify-content-between">
+                            <VIcon
+                                :color="visitor.color"
+                                small
+                            >
+                                mdi-circle
+                            </VIcon>
+                            <VLabel>{{ visitor.title }}</VLabel>
+                            <VLabel>
+                                <VIcon
+                                    :color="visitor.color"
+                                    small
+                                >
+                                    mdi-arrow-up
+                                </VIcon>
+                                {{ visitor.percentageDiff }}%
+                            </VLabel>
+                        </VRow>
+                    </VListItem>
+                </VList>
+            </VFlex>
+        </VLayout>
+    </VContainer>
+</template>
+
+<script>
+    import {GChart} from 'vue-google-charts';
+
+    export default {
+        components: {
+            GChart
+        },
+        props: {
+            data: {
+                type: Array,
+                required: true,
+                default: () => []
+            },
+            pieHole: {
+                type: Number,
+                default: 0.95
+            },
+            legend: {
+                type: Object,
+                required: true,
+                default: () => [],
```

Comment:
same as prev required prop, default is redundant, also next time take in account that `[]` is not quite same wuth type `Object`

Reply:
Fixed